tutorials/dapt-curation/code/modifiers.py

```diff
@@ -0,0 +1,38 @@
+# +
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -
+
+from nemo_curator.modifiers import DocumentModifier
+
+
+class QuotationUnifier(DocumentModifier):
```

Comment:
Looks like this is duplicated? There is another version inside `utils.py`. Let's delete this file, and keep the other file, given that all the other filters are also in that file.

Reply:
done